### DISCUSSION TITLE
Why does this constraint problem arise?

### DISCUSSION THREAD

**Discussion by 281674017** [yalmip/YALMIP#1456] [General discussion] [answered]
![1684d546dbb378e47b67dc77c6068dfe](https://github.com/user-attachments/assets/eca4bb1f-8dcc-4752-8d6a-d8239a2e4175)
![46f2110a1aa03a91534606790e03e663](https://github.com/user-attachments/assets/0278d13e-f224-4b47-82f6-f58d3cc3c59a)
Dear Professor, may I ask why setting the upper and lower limits to 2 or 3 does not return an error, but setting it to 1 does instead?

**Comment by johanlofberg** [comment, marked as answer]
Unless something weird happens when you divide by baseMWA (infinity, nans, zeros etc) your model simply appear to be really badly conditioned and you might want to try another solver to see if that helps, or follow the suggestion from gurobi about changing parameters, and most importantly check the numerics of the model and derive a better model. Finally, the solver is confused about whether it is possibly infeasible or unbounded, so the trivial first step is to remove the objective to settle that
